DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/AdminFields/RemoveAdminField.cshtml

```diff
@@ -11,7 +11,7 @@
   ViewData["HeaderPathName"] = "Tracking System";
   var confirmError = ViewData.ModelState[nameof(RemoveAdminFieldViewModel.Confirm)]?.Errors?.Count > 0;
   var confirmFormErrorClass = confirmError ? "nhsuk-form-group--error" : "";
-  var cancelLinkRouteData = new Dictionary<string, string> { { "customisationId", Model.CustomisationId.ToString() } };
+  var cancelLinkRouteData = new Dictionary<string, string> { { "customisationId", ViewContext.HttpContext.Request.RouteValues["customisationId"].ToString()! } };

```

Comment:
I think it'd be nice to pull ViewContext.HttpContext.Request.RouteValues["customisationId"].ToString() into a var or, even better, do something like:
var cancelLinkRouteData = ViewContext.HttpContext.Request.RouteValues.ToStringDictionary();

(we'd need an extension method to turn the route data into a string dictionary though)
e.g. something like here: https://github.com/aspnet/Mvc/issues/6358

Reply:
Created an Html helper extension GetRouteValues and used it for all the views' cancel link data